### PR TITLE
SOC-1508 | Replacing delete / undelete icon with more icon

### PR DESCRIPTION
**Pull request by bkoval**
https://wikia-inc.atlassian.net/browse/SOC-1508

@kvas-damian @rafalkalinski @Bloodmoon24 @katanka 

I'm leaving the whole Delete / Undelete component with SVG's as we'll just convert it into a button used in a popup. SVG-s stay too as without them the .hbs will break, let's remove them when we change the delete / undelete component into a menu item.

### CODE REVIEW

File: front/styles/main/module/_discussion.scss [hunk 1]
```diff
@@ -56,17 +56,25 @@ body.discussions {
 		}
 	}
 
+	.discussion-post-more-options {
+		display: inline-block;
+		float: right;
+		height: 16px;
+		text-align: right;
+		width: 16px;
```
Comment (bkoval) [inline front/styles/main/module/_discussion.scss]: I would keep the square dimensions on the wrapper to make sure the whole area is easily clickable.

File: front/templates/main/components/post-detail.hbs [hunk 2]
```diff
@@ -7,11 +7,9 @@
 	<span class="timestamp">
 		&bull; {{time-ago post.creationDate.epochSecond}}
 	</span>
-	{{discussion-delete
-		delete=this.attrs.deletePost
-		undelete=this.attrs.undeletePost
-		post=post
-	}}
+	<div class="discussion-post-more-options" {{action 'showMoreOptions'}}>
+		{{svg "more" viewBox="0 0 4 18" class="icon more-icon"}}
```
Comment (rafalkalinski) [inline front/templates/main/components/post-detail.hbs]: single quotes maybe?

**Comment by rafalkalinski** [comment]
LGTM

File: front/scripts/main/components/post-detail.js [hunk 3]
```diff
@@ -73,6 +73,14 @@ export default Ember.Component.extend(
 			cancelHideShareComponent() {
 				Ember.run.cancel(this.hideShareTimeout);
 			},
+
+			/**
+			 * This method will be used to show popup menu
```
Comment (kvas-damian) [inline front/scripts/main/components/post-detail.js]: I'm not sure if it's true...

This is how pop-over component define how to open:
https://github.com/Wikia/mercury/blob/89d74a51184a49b21174d4ceb23c26c4a24303a1/front/main/app/templates/components/post-detail.hbs#L60-L62

File: front/styles/main/module/_discussion.scss [hunk 4]
```diff
@@ -56,17 +56,25 @@ body.discussions {
...
+
+		.more-icon {
+			height: 16px;
```
Comment (kvas-damian) [inline front/styles/main/module/_discussion.scss]: This svg has 18px height, don't you think it'll break aspect ratio?

Comment (bkoval) [inline front/styles/main/module/_discussion.scss]: These dimensions are taken from zeplin, here: https://app.zeplin.io/project.html#pid=55de3ff843e8525d12c4a6af&sid=55e885a592a1d15b196e86ec

Comment (bkoval) [inline front/styles/main/module/_discussion.scss]: Let's discuss this with UX.

**Comment by hakubo** [comment]
lgtm

**Pull request closed without merging**